### PR TITLE
fix(containers): fix sort in the list when case differ

### PR DESCRIPTION
**Pull request by talend-sromain**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

When sort criteria contains elements on uppercase & others on lowercase, the natural order of the list is broken

**What is the new behavior?**

the list is sort without considering the case of items

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://247.talend.surge.sh)

File: packages/containers/src/List/selector.js [hunk 1]
```diff
@@ -78,10 +78,12 @@ export function configureGetFilteredItems(configure) {
 						if (a.get(sortBy).localCompare) {
 							return a.get(sortBy).localeCompare(b.get(sortBy));
 						}
-						if (a.get(sortBy) < b.get(sortBy)) {
+						const aValue = (a.get(sortBy) === null) ? '' : `${a.get(sortBy)}`.toLowerCase();
```
Comment (frassinier) [inline packages/containers/src/List/selector.js]:
```javascript
const aValue = `${a.get(sortBy) || ''}`.toLowerCase();
```

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://247.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://247.talend.surge.sh)

**Pull request merged**